webpack/landing.js

```diff
@@ -36,6 +37,7 @@ ReactDOM.render(
           <Route path='/questions/new' component={QuestionEditContainer} />
           <Route path='/questions/:qId' component={QuestionShowContainer} />
           <Route path='/questions/:qId/:action' component={QuestionEditContainer} />
+        <Route path='/survey/:surveyId' component={SurveyShowContainer} />
```

Comment:
pretty much my only feedback is should this be plural to match the other routes? Otherwise all tests are passing and its ready to merge